src/handlers/motions/motionFinalized/helpers.ts

```diff
@@ -440,7 +439,7 @@ export const claimExpenditurePayouts = async (
   // - The value is not 0
   // - The storage slot is not 26
   // Then we can assume this state change ain't a stage release
-  if (convertedStorageSlot !== 26 || convertedValue !== 0) {
+  if (convertedStorageSlot !== 26 || BigNumber.from(0).eq(value)) {
```

Comment:
Just a small fix to check the value is _not_ 0:
```suggestion
  if (convertedStorageSlot !== 26 || !BigNumber.from(0).eq(value)) {
```

Reply:
🤦‍♂️ Fix is up now!